src/libraries/System.Runtime/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -17,4 +17,21 @@ MembersMustExist : Member 'public System.Runtime.Serialization.DeserializationTo
 MembersMustExist : Member 'public void System.Runtime.Serialization.SerializationInfo.ThrowIfDeserializationInProgress()' does not exist in the reference but it does exist in the implementation.
```

Comment:
These baselines may deserve another entry in https://github.com/dotnet/runtimelab/issues/52

Reply:
ok